controller_manager:
  ros__parameters:
    update_rate: 100
    use_sim_time: true

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

joint_state_broadcaster:
  ros__parameters:
    use_sim_time: true

joint_trajectory_controller:
  ros__parameters:
    use_sim_time: true
    joints:
      - hip1_fl
      - hip2_fl
      - knee_fl
      - hip1_fr
      - hip2_fr
      - knee_fr
      - hip1_bl
      - hip2_bl
      - knee_bl
      - hip1_br
      - hip2_br
      - knee_br
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity
